packages/commonwealth/client/scripts/views/components/component_kit/new_designs/CWTable/useCWTableState.tsx

```diff
@@ -0,0 +1,53 @@
+import { APIOrderDirection } from 'client/scripts/helpers/constants';
+import { useMemo, useState } from 'react';
+import { CWTableColumnInfo, CWTableSorting } from './CWTable';
+
+type UseCWTableStateProps = {
+  columns: CWTableColumnInfo[];
+  initialSortColumn?: string;
+  initialSortDirection: APIOrderDirection;
+};
+
+export type CWTableState = {
+  columns: CWTableColumnInfo[];
+  orderBy: string;
+  orderDirection: string;
+  sorting: CWTableSorting;
+  setSorting: (newSorting: CWTableSorting) => void;
+};
+
+export function useCWTableState({
```

Comment:
please use const instead of function

Reply:
Please see `Other Considerations` in the PR description– those components were never "fixed" to use the new changes.

Please list instructions for how to access those components in the UI.

Reply:
The CWTable component still utilizes internal sorting btw. That was never removed.

Reply:
1. directory page
  - be an admin
  - go to integrations
  - enable directory page
  - click go to directory page

https://github.com/hicommonwealth/commonwealth/assets/14819225/23ed3def-7f97-4164-b762-be8ef7eed25c

2. components showcase
  - http://localhost:8080/components => tables section

3. my community stake / transactions are behind feature flag `FLAG_MY_COMMUNITY_STAKE_PAGE_ENABLED` and then available in the user dropdown in top navbar cc @mzparacha